### PR TITLE
Feature/global exception handling

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: molgenis-data-cache/src/main/java/org/molgenis/data/cache/l2/L2Cache.java [hunk 1]
```diff
@@ -115,11 +114,7 @@ public Entity get(Repository<Entity> repository, Object id)
 		catch (ExecutionException exception)
 		{
 			// rethrow unchecked
-			if (exception.getCause() != null && exception.getCause() instanceof RuntimeException)
-			{
-				throw (RuntimeException) exception.getCause();
-			}
-			throw new MolgenisDataException(exception);
+			throw new RuntimeException(exception);
```
Comment (fdlk) [inline molgenis-data-cache/src/main/java/org/molgenis/data/cache/l2/L2Cache.java]: Why this change?

File: molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LocalizationService.java [hunk 2]
```diff
@@ -163,15 +162,10 @@ public void addMissingMessageIDs(String namespace, Set<String> messageIDs)
...
-				LOG.debug("Added message IDs to namespace '{}' : {}.", namespace, messageIDs);
-			}
-			catch (MolgenisPermissionException ex)
-			{
-				LOG.info("No permission to add message IDs to namespace '{}' : {}.", namespace, messageIDs, ex);
-			}
+
+			dataService.add(L10N_STRING, entities);
+			LOG.debug("Added message IDs to namespace '{}' : {}.", namespace, messageIDs);
+
```
Comment (fdlk) [inline molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LocalizationService.java]: If we remove the catch, the method will suddenly begin to throw runtime exceptions it previously nommed up. Is this intentional?

File: molgenis-data-cache/src/main/java/org/molgenis/data/cache/l2/L2Cache.java (hunk 1 above)
Comment (bartcharbon) [reply molgenis-data-cache/src/main/java/org/molgenis/data/cache/l2/L2Cache.java]: MolgenisDataException is deprecated, it's not something to show to the user, so RuntimeException would be fine I think

File: molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LocalizationService.java (hunk 2 above)
Comment (bartcharbon) [reply molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LocalizationService.java]: Ahum.... cannot remember removing this... will revert
